themes/bootstrap3/templates/record/checkbox.phtml

```diff
@@ -1,6 +1,6 @@
+<?php $label = isset($this->context) ? 'select_item_' . $this->context : 'select_item'; ?>
 <label class="record-checkbox hidden-print">
-  <input class="checkbox-select-item" type="checkbox" name="ids[]" value="<?=$this->escapeHtmlAttr($this->id) ?>"<?php if (isset($this->formAttr)): ?> form="<?=$this->formAttr ?>"<?php endif; ?>>
+  <input class="checkbox-select-item" type="checkbox" name="ids[]" value="<?=$this->escapeHtmlAttr($this->id) ?>"<?php if (isset($this->formAttr)): ?> form="<?=$this->formAttr ?>"<?php endif; ?> aria-describedby="<?=$this->escapeHtmlAttr($this->id) ?>" aria-label="<?=$this->transEscAttr($label)?>">
```

Comment:
I believe `aria-label` and `aria-describedby` [won't interfere](https://www.w3.org/TR/accname-1.2/#dfn-accessible-description), but I don't know for sure. I would expect this to announce to a screen-reader: "Select Item. Checkbox. {Record Title}."

Reply:
The {Record Title} is read by a screen reader, in the list of results and the other lists where the checkboxes are used.
Orca is reading for example: "list with one item. checkbox not checked. {Record Title}."